BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Microsoft.ApplicationInsights.Tests.csproj

```diff
@@ -22,6 +22,7 @@
     <PackageReference Include="Moq" Version="4.10.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="2.2.3" />
     <PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
Why do we need this package reference?

Reply:
We have a handful of unit tests that are deserializing json.
This project has always had a dependency on Newtonsoft, but previously it was an implicit dependency via Bond.